src/battle-tooltips.ts

```diff
@@ -663,41 +662,17 @@ class BattleTooltips {
 		}
 
 		const supportsAbilities = this.battle.gen > 2 && !this.battle.tier.includes("Let's Go");
+		text += '<p>';
+		if (supportsAbilities) {
+			text += this.getPokemonAbilityText(clientPokemon, serverPokemon);
+		}
 		if (serverPokemon) {
-			if (supportsAbilities) {
-				let abilityText = Dex.getAbility(serverPokemon.baseAbility).name;
-				let ability = Dex.getAbility(serverPokemon.ability || pokemon.ability).name;
-				if (ability && (ability !== abilityText)) {
-					abilityText = ability + ' (base: ' + abilityText + ')';
-				}
-				text += '<p><small>Ability:</small> ' + abilityText;
-				if (serverPokemon.item) {
-					text += ' / <small>Item:</small> ' + Dex.getItem(serverPokemon.item).name;
-				}
-				text += '</p>';
-			} else if (serverPokemon.item) {
+			if (serverPokemon.item) {
 				let itemName = Dex.getItem(serverPokemon.item).name;
-				text += '<p><small>Item:</small> ' + itemName + '</p>';
+				// ability/item on one line for your own tooltips
+				text += '/ <small>Item:</small> ' + itemName;
```

Comment:
Non-base abilities shouldn't be shown on the switch list, probably. And this shouldn't be a one-liner if you have an active Pokémon... I would detect `isActive`.

Like, if `isActive` is false, don't call `getPokemonAbilityText` at all, and put the ability/item on the same line. Otherwise, do call `getPokemonAbilityText` and put the item on a separate line.

Reply:
So the behavior should be changed from what it currently does? Because currently it shows ability/item in one line even for the active mon.

Non-base abilities already don't show up in the switch list, not because of anything this function is doing but because the temporary ability changes are getting properly cleaned up on switchout I imagine.
I'd prefer to avoid a situation where we have a dedicated function for something and then we bypass it sometimes.

Reply:
Yeah, I wouldn't mind changing the behavior slightly.

Also: I was referring to non-base abilities for the active Pokémon in the switch list.

Reply:
Is there a way to distinguish between "switch tooltip for active pokemon" and "battle mouseover tooltip of active pokemon"? I was under the impression that there wasn't right now. Because the latter definitely needs to show the non-base ability.
But I also don't really understand what the point of this is in the first place. The base ability is being displayed in parens already, what's to gain from hiding the non-base one? Do people get confused over this?

Reply:
> Is there a way to distinguish between "switch tooltip for active pokemon"

Yes, `isActive` will be `false` in this case.

> and "battle mouseover tooltip of active pokemon"?

And `isActive` will be `true` in this case.

> But I also don't really understand what the point of this is in the first place. The base ability is being displayed in parens already, what's to gain from hiding the non-base one? Do people get confused over this?

Showing only the base ability makes it consistent with other Pokémon in the switch menu. It's mostly to keep switch menu tooltips consistent, which should make them easier to read.